.circle/pack.yaml.schema

```diff
@@ -8,6 +8,7 @@ properties:
     type: array
   version:
     type: string
+    pattern: "^(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-[\da-z\-]+(?:\.[\da-z\-]+)*)?(?:\+[\da-z\-]+(?:\.[\da-z\-]+)*)?$"
```

Comment:
Oh, no, it's a regex! 

LGTM, but it would be good to describe valid values in `description` attribute so it's more immediately obvious what are the valid values :)